### DISCUSSION TITLE
About template update

### DISCUSSION THREAD

**Discussion by heipizhu4** [carriontrooper/Alpha-Frontier#38] [Ideas]
Who will be responsible for updating the `template.properties`? Me or @carriontrooper when he add something requires translation or @ArchDuque-Pancake ?

**Comment by Caballero-Arepa** [comment]
It should be me, is there a new string that isnt translated?

**Reply by heipizhu4** [reply]
More strings in f31be740a786635ed9117de2aaaeaf0af381a95e & 8d5a33530753d626d4d39f46f7feba6de74348e2 .
Strings changed in e481b81442c67c292cbc41cf118c7ccaf811fa7f & 8a82dadf923926754832ca312c60a6bbd7261c6b .

**Reply by Caballero-Arepa** [reply]
Thanks, I'll get on it
